setup.py

```diff
@@ -278,12 +278,14 @@ def build_extension(self, ext):
     # https://www.pyopenssl.org/en/stable/api/ssl.html#OpenSSL.SSL.Context.set_default_verify_paths
     pyopenssl_reqs.append("certifi")
 
+aws_reqs = ["pymongo-auth-aws<2.0.0"]
+
 extras_require = {
-    "encryption": ["pymongocrypt>=1.3.0,<2.0.0"],
+    "encryption": ["pymongocrypt>=1.3.0,<2.0.0"] + aws_reqs,
     "ocsp": pyopenssl_reqs,
     "snappy": ["python-snappy"],
     "zstd": ["zstandard"],
-    "aws": ["pymongo-auth-aws<2.0.0"],
+    "aws": aws_reqs,
```

Comment:
Let's update the docs to say that pymongo-auth-aws is also required for encryption in these 3 places:
```
README.rst:135:  $ python -m pip install "pymongo[encryption]"
doc/examples/encryption.rst:30:  $ python -m pip install 'pymongo[encryption]'
doc/installation.rst:75:  $ python3 -m pip install "pymongo[encryption]"
```

Reply:
Isn't the main point of the extra that the user doesn't need to concern themselves with the libraries used?

Reply:
Sure but we should still document what libraries will be installed.

Reply:
Note I'm not saying to change those 3 lines in particular. Just the surrounding test. For example the readme says:
```
Client-Side Field Level Encryption requires `pymongocrypt
<https://pypi.org/project/pymongocrypt/>`_::

  $ python -m pip install "pymongo[encryption]"

```
But this is now incomplete because it doesn't mention pymongo-auth-aws.

Reply:
Fixed